app/services/support_interface/qualifications_export.rb

```diff
@@ -0,0 +1,144 @@
+          'GCSE English studies single grade' => english_structured_gcse_grades(qualifications, 'english_studies_single_award'),
+          'GCSE English studies double grade' => english_structured_gcse_grades(qualifications, 'english_studies_double_award'),
+          'GCSE English other grade' => english_other_gcse_grade(qualifications),
+
+          'A level 1 subject' => a_levels[0].try(:subject),
+          'A level 1 grade' => a_levels[0].try(:grade),
+          'A level 2 subject' => a_levels[1].try(:subject),
+          'A level 2 grade' => a_levels[1].try(:grade),
+          'A level 3 subject' => a_levels[2].try(:subject),
+          'A level 3 grade' => a_levels[2].try(:grade),
+          'A level 4 subject' => a_levels[3].try(:subject),
+          'A level 4 grade' => a_levels[3].try(:grade),
+          'A level 5 subject' => a_levels[4].try(:subject),
+          'A level 5 grade' => a_levels[4].try(:grade),
+
+          'Degree 1 type' => degrees[0].try(:qualification_type),
+          'Degree 1 grade' => degrees[0].try(:grade),
+          'Degree 2 type' => degrees[1].try(:qualification_type),
+          'Degree 2 grade' => degrees[1].try(:grade),
+
+          'Number of other qualifications provided' => other_qualification_count(qualifications),
+        }
+        output
+      end
+    end
+
+  private
+
+    def maths_gcse_grade(qualifications)
+      maths_gcse = qualifications.where(level: :gcse, subject: :maths).first
+      maths_gcse.try(:grade)
+    end
+
+    def science_single_gcse_grade(qualifications)
+      unstructured_grade = qualifications.where(level: :gcse, subject: :science).first.try(:grade)
+      return unstructured_grade if unstructured_grade.present? && SINGLE_GCSE_GRADES.include?(unstructured_grade)
+
+      science_single_gcse = qualifications.where(level: :gcse, subject: ApplicationQualification::SCIENCE_SINGLE_AWARD).first
+      science_single_gcse.try(:grade)
+    end
+
+    def science_double_gcse_grade(qualifications)
+      unstructured_grade = qualifications.where(level: :gcse, subject: :science).first.try(:grade)
+      return unstructured_grade if unstructured_grade.present? && DOUBLE_GCSE_GRADES.include?(unstructured_grade)
+
+      science_double_gcse = qualifications.where(level: :gcse, subject: ApplicationQualification::SCIENCE_DOUBLE_AWARD).first
+      science_double_gcse.try(:grade)
+    end
+
+    def science_triple_gcse_grade(qualifications)
+      unstructured_grade = qualifications.where(level: :gcse, subject: :science).first.try(:grade)
+      return unstructured_grade if unstructured_grade.present? && not_single_or_double_award?(unstructured_grade)
+
+      science_triple_gcse = qualifications.where(level: :gcse, subject: ApplicationQualification::SCIENCE_TRIPLE_AWARD).first
+      return if science_triple_gcse.try(:structured_grades).blank?
+
+      science_triple_gcse[:structured_grades].values.join
+    end
+
+    def english_unstructured_gcse_grade(qualifications)
```

Comment:
Same with this in terms of granularity. 

I'd lean towards keeping it as is as you've already done a great job with all the legwork, but if Mike won't use it we can just do 

`def english_gcse_grade(application_form)
  english_gcse = application_form.english_gcse
  JSON.parse(english_gcse.structured_grades)&.values&.join || english_gcse.grade
end`

Reply:
Similarly, this was a request from Mike